src/xcode/ENA/ENAUITests/DeltaOnboarding/ENAUITestsDeltaOnboarding.swift

```diff
@@ -73,22 +73,6 @@ class ENAUITests_06_DeltaOnboarding: XCTestCase {
 		app.swipeUp()
 	}
 	
-	func test_DeltaOnboardingScreenCloseButton() throws {
```

Comment:
Would it be easier to disable the test in the Testplan? 🤔 How do we remind ourselves that we have to fix this?